lua/starfall/sflib.lua

```diff
@@ -1210,7 +1210,7 @@ end
 function SF.CheckModel(model, player)
 	if #model > 260 then return false end
 	model = SF.NormalizePath(string.lower(model))
-	if util.IsValidModel(model) and util.IsValidProp(model) then
+	if string.GetExtensionFromFilename(model) == "mdl" and (CLIENT or (util.IsValidModel(model) and util.IsValidProp(model))) then
```

Comment:
Need to make sure the IsValidProp doesn't break some hologram cases. The wiki provides no info of what this function does

Reply:
A valid prop is most likely one that has physics, so it will need to be moved out of this function.